.github/workflows/build-triton-wheel.yml

```diff
@@ -90,7 +94,7 @@ jobs:
 
           BUILD_ROCM=""
           if [[ "$BUILD_DEVICE" == "rocm" ]]; then
-            BUILD_ROCM="--build-rocm"
+            BUILD_ROCM="--build-rocm --rocm_version ${{ matrix.rocm_version }}"
```

Comment:
Here try to pass ${{ matrix.rocm_version } as ENV variable and then use it like this:
```
 BUILD_ROCM="--build-rocm --rocm_version ${ROCM_VERSION}"
```

Reply:
Thank you @atalman I have made the suggested changes and ensured on CUDA matrix.rocm_version is set to empty string to avoid any complications.

Unfortunately I am still running into a lintrunner issue that I cannot find the cause of
```
2023-08-24T10:45:47.1602107Z >>> Lint for .github/workflows/build-triton-wheel.yml:
2023-08-24T10:45:47.1602297Z 
2023-08-24T10:45:47.1602642Z   Error (ACTIONLINT) [shellcheck]
2023-08-24T10:45:47.1602942Z     shellcheck reported issue in this script: SC2086:info:39:106: Double quote
2023-08-24T10:45:47.1603315Z     to prevent globbing and word splitting
2023-08-24T10:45:47.1603471Z 
2023-08-24T10:45:47.1603664Z          62  |          docker-image: ${{ env.DOCKER_IMAGE }}
2023-08-24T10:45:47.1603919Z          63  |
2023-08-24T10:45:47.1604197Z          64  |      - name: Build Triton wheel
2023-08-24T10:45:47.1604476Z     >>>  65  |        run: |
2023-08-24T10:45:47.1604722Z          66  |          set -x
2023-08-24T10:45:47.1605039Z          67  |          mkdir -p "${RUNNER_TEMP}/artifacts/"
2023-08-24T10:45:47.1605374Z          68  |          container_name=$(docker run \
```

It seems to come from this line
`BUILD_ROCM="--build-rocm --rocm_version ${ROCM_VERSION}"`

Does anything jump out to you here?

Reply:
Removing this ROCM_VERSION as it's causing us headaches with lintrunner and it is non essential to pass to the build script. We only need reference to the matrix.rocm_version to set the correct DOCKER_IMAGE